src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -2471,7 +2461,7 @@ bc42025]]>,
         Dim tree = ParseAndVerify(<![CDATA[
 Module Module1
     Sub Main
-#enable warning BC42025, someid, 
```

Comment:
It looks like many whitespace-only changes in this file, along with one or two actual changes. Please revert to only do the intended change (whitespaces should probably be left alone here, or addressed separately if not).